roles/servicetelemetry/templates/manifest_grafana.j2

```diff
@@ -4,23 +4,65 @@ metadata:
   name: {{ meta.name }}
   namespace: {{ meta.namespace }}
 spec:
+  serviceAccount:
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirectreference.primary: '{{ grafana_oauth_redir_ref | to_json }}'
   baseImage: {{ servicetelemetry_vars.graphing.grafana.base_image }}
   ingress:
     enabled: {{ servicetelemetry_vars.graphing.grafana.ingress_enabled }}
+    targetPort: web
+    termination: reencrypt
+  client:
+    preferService: true
   config:
     auth:
-      disable_signout_menu: {{ servicetelemetry_vars.graphing.grafana.disable_signout_menu }}
+      disable_signout_menu: true
+      disable_login_form: true
```

Comment:
I don't think I remember seeing the `disableSignoutMenu` parameter in the ServiceTelemetry CRD removed. Should probably clean that up.